osu.Game/Screens/Play/Player.cs

```diff
@@ -611,6 +634,15 @@ private void updateCompletionState(ValueChangedEvent<bool> completionState)
                 return score.ScoreInfo;
             });
 
+            var storyboardHasOutro = DimmableStoryboard.ContentDisplayed && (!DimmableStoryboard.HasStoryboardEnded?.Value ?? false);
+
+            if (storyboardHasOutro)
+            {
+                skipOutroOverlay.Show();
+                completionProgressDelegate = null;
+                return;
+            }
```

Comment:
```suggestion
            if (DimmableStoryboard.ContentDisplayed &&
                DimmableStoryboard.HasStoryboardEnded?.Value == false)
            {
                skipOutroOverlay.Show();
                return;
            }
```

Reply:
Disagree with inlining the conditional. Variable is fine. Just discarding the delegate without cancellation is extremely dicey, though.

Reply:
The delegate should not be non-null at this point, that's why I removed it in my suggestion, to avoid confusion.